packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-title-indicator.tsx

```diff
@@ -26,7 +26,7 @@ const JurisdictionTitleIndicator = ({
 
     const getVerificationIconVariant = (verification_document: TJurisdictionCardItemVerificationItem): string => {
         let icon_variant: TJurisdictionCardVerificationStatus = 'Default';
-        if (type_of_card === Jurisdiction.BVI || type_of_card === Jurisdiction.LABUAN) {
+        if ([Jurisdiction.BVI, Jurisdiction.VANUATU, Jurisdiction.LABUAN].includes(type_of_card)) {
```

Comment:
why are you getting rid of BVI?

Reply:
@matin-deriv as per the new figma design, now the grouping is (BVI, VANUATU, LABUAN), and (MALTA) alone.
Earlier it was (BVI, LABUAN), and (VANUATU, MALTA).
![Screenshot 2023-05-17 at 11 24 29 AM](https://github.com/amina-deriv/deriv-app/assets/122449658/e8c581f8-afe6-4e33-8584-e321b3a57193)